test/vector_of_arrays.jl

```diff
@@ -97,8 +97,45 @@ using ArraysOfArrays: full_consistency_checks, append_elemptr!
         @test vcat(B1, B2, B3, B4) == vcat(A1, A2, A3, A4)
         @test eltype(vcat(B1, B2, B3, B4)) == Array{Float64,3}
         full_consistency_checks(vcat(B1, B2, B3, B4))
+
+        B1_copy = @inferred(copy(B1)); B3_copy = @inferred(copy(B3))
+        append!(B1_copy, B3_copy)
+        @test B1_copy.data == vcat(B1.data, B3.data)
+
     end
 
+    @testset "indexing" begin
+        V1 = @inferred(VectorOfArrays(ref_AoA3(Float32, 3)))
+        V2 = @inferred(VectorOfArrays(ref_AoA3(Float32, 3)))
+        V12 = vcat(V1, V2)
+        ind_style = @inferred(IndexStyle(V12))
+        @test ind_style == IndexLinear()
+        for i in 1:length(V12)
+            @test getindex(V12, i) == V12[i]
+        end
+        @test getindex(V12, 1:length(V12)) == V12
+        
+        @test @inferred(element_ptr(V12)) == V12.elem_ptr
+
+## _view_reshape_spec not yet implemented ##
+#       V1_copy = copy(V1)
+#       V2_copy = copy(V2)
+#       @test setindex!(V1_copy, V1, 1) == V1
+#       setindex!(V2_copy, V2, 1)
+#       @test V2_copy[1] == V2_copy[2]
+
+## function mul(s) not yet implemented ##
+#       sizehint!(v12_copy, 2, (2,2,3))
```

Comment:
What are these two missing functions supposed to do? Should I try to implement them?

Reply:
No, not yet.